pkg/agent/police.go

```diff
@@ -753,6 +769,52 @@ func buildMangleStaticRule(base uint32) map[string][]iptables.Rule {
 	return res
 }
 
+func buildReplyRouteIptables(base uint32, dev string) map[string][]iptables.Rule {
+	res := map[string][]iptables.Rule{
+		"PREROUTING": {
+			{
+				Match:  iptables.MatchCriteria{},
+				Action: iptables.JumpAction{Target: "EGRESSGATEWAY-REPLY-ROUTING"},
+				Comment: []string{
+					"It's from EgressGetaway",
```

Comment:
EgressGetaway reply packet rule for underlay cni

Reply:
> EgressGetaway reply packet rule for underlay cni

done